README.md

```diff
@@ -104,16 +103,34 @@ does. Run it against a VM and make sure you're happy. ***Do not*** blindly
 download code from the internet and trust that it's going to work as you expect.
 :skull:
 
-1. Enable the Ubuntu Universe repository: `sudo add-apt-repository universe`
-2. Install Ansible: `sudo apt install ansible`
-3. `git clone https://github.com/davestephens/ansible-nas.git && cd ansible-nas`
-4. Copy `group_vars/all.yml.dist` to  `group_vars/all.yml`.
-5. Open up `group_vars/all.yml` and follow the instructions there for configuring your Ansible NAS.
-6. If you plan to use Transmission with OpenVPN, also copy `group_vars/vpn_credentials.yml.dist` to
-`group_vars/vpn_credentials.yml` and fill in your settings.
-7. Copy `inventory.dist` to `inventory` and update it.
-8. Install the dependent roles: `ansible-galaxy install -r requirements.yml` (you might need sudo to install Ansible roles)
-9. Run the playbook - something like `ansible-playbook -i inventory nas.yml -b -K` should do you nicely.
+
+1. Enable the Ubuntu Universe repository:
+
+    `sudo add-apt-repository universe`
+
+2. Install Ansible:
+
+    `sudo apt install ansible`
+
+3. Clone Ansible-NAS:
+
+    `git clone https://github.com/davestephens/ansible-nas.git && cd ansible-nas`
+
+4. Create your own inventory and config files by copying `inventories/sample` to your own directory:
+
+    `cp -rfp inventories/sample inventories/my-ansible-nas`
+
+5. Review `group_vars/all.yml`. Change settings by overriding them in `inventories/my-ansible-nas/group_vars/all.yml`.
+
+6. Update `inventories/my-ansible-nas/inventory`.
+
+7. Install the dependent roles: `ansible-galaxy install -r requirements.yml` (you might need `sudo` to install Ansible roles).
+
+8. Run the playbook - something like `ansible-playbook -i inventory nas.yml -b -K` should do you nicely.
```

Comment:
Should the playbook run command reference `inventories/my-ansible-nas` instead of just `inventory` now?

`ansible-playbook -i inventories/my-ansible-nas nas.yml -b -K`

Reply:
Yes, good spot! Fixed.